xtensor-signalConfig.cmake.in

```diff
@@ -6,16 +6,42 @@
 # The full license is in the file LICENSE, distributed with this software. #
 ############################################################################
 
-# xtensor-python cmake module
+# xtensor cmake module
 # This module sets the following variables in your project::
 #
-#   xtensor-signal_FOUND - true if xtensor-python found on the system
-#   xtensor-signal_INCLUDE_DIRS - the directory containing xtensor-python headers
-#   xtensor-signal_LIBRARY - empty
+#   xtensor_FOUND - true if xtensor found on the system
+#   xtensor_INCLUDE_DIRS - the directory containing xtensor headers
+#   xtensor_LIBRARY - empty
```

Comment:
The original values were correct. `xtensor_XXX` is used by the xtensor "core" package.